packages/utils/src/lib/string.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Formats a price string to a specified number of decimal places.
+ * If the input price is undefined or cannot be parsed as a number,
+ * it defaults to 0. The function returns the formatted price as a string.
+ *
+ * @param {string} [price] - The price to be formatted. Optional.
+ * @param {number} [decimals=2] - The number of decimal places to format the price to. Defaults to 2.
+ * @returns {string} - The formatted price.
+ */
+export const formatPriceToDecimal = (price?: string, decimals = 2): string => {
+  const numericPrice = price ? parseFloat(price) : 0;
```

Comment:
Should this default to 0? This might cause issues as a receipt showing 0 when they get charged with an appropriate Ada value? Or maybe it just gets handled in the `PriceSummaryDialog` to an Error, contact support, or unavailable?